app/src/test/java/com/example/gymbuddy/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.gymbuddy
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
+        assertEquals(4, 2 + 2)
+    }
+}
```

Comment:
we can probably get rid of this class since it is example code, that way we can save on space and potential glitches